ampersand-filter-tracker.js

```diff
@@ -10,7 +10,8 @@
       filter: 'function'
     },
     session: {
-      handles: 'array'
+      handles: 'array',
+      onApply: [ 'function', false, function() { return function() {}; } ]
```

Comment:
Will definitely do the `_.noop`. And the function returning a function is just an ampersand limitation. If the default value is a function, ampersand assumes you want to call it to return an object or something fancy as the default.